python/whylogs/core/validators/validator.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+from typing import Any, Callable, Dict, List
+
+from whylogs.core.metrics.condition_count_metric import Condition
+from whylogs.core.preprocessing import PreprocessedColumn
+
+
+class Validator(ABC):
```

Comment:
This is ok, but do we need a base class? Is that intended to be an extensibility point?

Reply:
Yes, I thought we could extend from Base Validator class if we have Validators operating on Metrics. But if the Conditions themselves are changed to work with metrics, this use case might not be valid anymore.

But we could have other unforeseen validators in the future.

What do you think? Do we want to remove that base class, or keep it?

Reply:
I was looking to understand the motivation, thanks for explanation.